README.md

```diff
@@ -0,0 +1,61 @@
+# urdf-packager
+
+A small util to turn a robot model into a single self-contained archive.
+
+## Use
+
+### Env setup
+
+Create a ROS 2 ready environment:
+
+```bash
+lxc launch ubuntu:22.04 urdf-exporter --config=user.user-data="$(curl -L https://gist.githubusercontent.com/artivis/1fcfcc38f1cb8c087e27ebda5484f2ef/raw/ros-bare.cloud-init.yaml)"
+```
```

Comment:
Does the ROS 2 env creation belongs here?

Reply:
This examplify a complete workflow that doesn't pollute the host machine.
Thus this is just convenient.

Reply:
Between the shelling as ubuntu user and the launching of the container with lxc and a link from the internet, this just looks untidy. I would rather see a multipass vm that can be launched with a simple cli and shell into with the default ubuntu user.

Reply:
Users that does not want to pollute their system will know how. you could suggest a general advice to run everything inside a multipass vm at the beginning, but commands should focus on the tool, not its isolation from the host.